### DISCUSSION TITLE
How to create Quarto listing inside of HTML code ?

### DISCUSSION THREAD

**Discussion by alanmaulanaa** [quarto-dev/quarto-cli#9968] [Q&A]
### Description

So I'm trying to make grid card like this
https://codepen.io/jmegs/pen/opbyXg

And by using this 
````qmd
```{=html}
```
````
The HTML code and it's CSS works fine and now how to make it working using listing method ?
So far trying to ask using GPT but nothing's work.

Is there any guide for this ?

**Comment by mcanouil** [comment]
What do you mean "using listing"?
If your code is to customise the listing page, then see <https://quarto.org/docs/websites/website-listings-custom.html#listing-templates>.

> So far trying to ask using GPT but nothing's work.

Note that the page can be found using "listings" as keyword in the search. LLMs won't beat a simple search and reading of documentation, especially with recent tools.
![image](https://github.com/quarto-dev/quarto-cli/assets/8896044/e5a2aba2-294d-4b75-b6ca-cb48ebf41437)
